src/components/SightingPage.js

```diff
@@ -0,0 +1,113 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams, useNavigate, useOutletContext } from "react-router-dom";
+import { BACKEND_URL } from "../constants";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { Button, TextField } from "@mui/material";
+import "../App.css";
+
+export default function SightingPage() {
+  const params = useParams();
+  const id = params.sightingId;
+  const [sighting, setSighting] = useState();
+  const [comments, setComments] = useState();
+  const [content, setContent] = useState(null);
+  const [categories, setCategories] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const takeSightings = async () => {
+      try {
+        const sightingData = await axios.get(`${BACKEND_URL}/sightings/${id}`);
+        console.log(id);
+        const sightingInfo = sightingData.data;
+
+        setSighting(sightingInfo);
+
+        setCategories(sightingInfo.categories);
+
+        const commentData = await axios.get(
+          `${BACKEND_URL}/sightings/${id}/comments`
+        );
+        const commentInfo = commentData.data;
+        setComments(commentInfo);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    takeSightings();
+  }, [id]);
+
+  const categoriesList = categories.map((category) => (
+    <span>{category.name} </span>
+  ));
+
+  let sightingMoreInfo = null;
+  if (sighting) {
+    sightingMoreInfo = (
+      <div>
+        <Card className="page-layout">
+          <CardContent>
+            <Typography>{sighting.location}</Typography>
+            <Typography>{sighting.date}</Typography>
+            <Typography>{sighting.notes}</Typography>
+            <Typography>Categories: {categoriesList}</Typography>
+          </CardContent>
+        </Card>
+      </div>
+    );
+  }
+
+  let commentsList = null;
+  if (comments) {
+    commentsList = comments.map((comment) => {
+      return (
+        <Card>
+          <CardContent>
+            <Typography variant="body2">{comment.content}</Typography>
+          </CardContent>
+        </Card>
+      );
+    });
+  }
```

Comment:
I feel this is a bit redundant and could just be conditionally rendered based on comments and sighting in the return statement. Ideally you create components for those as JSX elements as well